### ISSUE TITLE
this._element.style is undefined

### ISSUE THREAD

**Issue by BrainFeeder** [nakupanda/bootstrap3-dialog#372]
I'm using the latest version, and the latest Bootstrap 4 release. If I try to use bootstrapDialog,
it triggers an error in console: TypeError: this._element.style is undefined

**Comment by centurianii** [comment]
This project is for bootstrap 3.

**Comment by BrainFeeder** [comment]
Yeah, I downloaded the compatible project for bs4 and posted the issue in the wrong repo :) Sorry.
